options/locale/locale_en-US.ini

```diff
@@ -1618,6 +1618,8 @@ pulls.reopened_at = `reopened this pull request <a id="%[1]s" href="#%[1]s">%[2]
 pulls.merge_instruction_hint = `You can also view <a class="show-instruction">command line instructions</a>.`
 pulls.merge_instruction_step1_desc = From your project repository, check out a new branch and test the changes.
 pulls.merge_instruction_step2_desc = Merge the changes and update on Gitea.
+pulls.clear_merge_message = Clear merge message
```

Comment:
IMO it's "reset merge message", but not "clear"

It's just my  thought,  not a blocker,  otherwise  L-G-T-M.